include/drivers/st25r3911b_nfc.rst

```diff
@@ -0,0 +1,84 @@
+.. _st25r3911b_nfc_readme:
+
+NFC Reader ST25R3911B
+#####################
+
+NFC Reader ST25R3911B can be used to read/write in ISO/IEC 14443 A compatible NFC tags,
+creating an NFC polling device. The library is intended to use with the NFC reader ST25R3911B
+Nucleo expansion board.
```

Comment:
The board name can be added here X-NUCLEO-NFC05A1 (if this is the correct code)

Reply:
added (here and in the sample)